### ISSUE TITLE
Local workbench is completely blank

### ISSUE THREAD

**Issue by hugoabernier** [SharePoint/sp-dev-docs#4261]
#### Category
- [ ] Question
- [ ] Typo
- [x] Bug
- [ ] Additional article idea

#### Expected or Desired Behavior

Local workbench should render some visible content and not produce any javascript errors.

#### Observed Behavior

I hate to post an issue here, but I hope that solving my issue will help someone else.

Several months ago, my local workbench stopped working completely. Since then, running `gulp serve` on any SPFx solution in *any browser* will render the following HTML:

```HTML
 <!doctype html>
  <html dir="ltr">
   <head>
   <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
   <meta http-equiv="X-UA-Compatible" content="IE=edge" />
   <meta name="viewport" content="width=device-width, initial-scale=1" />
    
   <title>SharePoint Web Part Workbench</title>
    
   <link rel="shortcut icon" href="https://localhost:4321/temp/workbench-packages/@microsoft_sp-webpart-workbench/lib/api/assets/server-icon.png" />
    
   <script type="text/javascript" src="https://localhost:4321/temp/manifests.js"></script>
   <script type="text/javascript" src="https://localhost:4321/temp/workbench-packages/@microsoft_sp-loader/dist/sp-loader-assembly_default.js"></script>
   <script type="text/javascript">
   // The workbenchInit.js script is commonJS and expects an "exports" object
   var exports = {};
    
   var syntheticManifests = [
   {
   "id": "0d910c1c-13b9-4e1c-9aa4-b008c5e42d7d",
   "manifestVersion": 2,
   "componentType": "Library",
   "alias": "react",
   "version": "16.7.0",
   "loaderConfig": {
   "internalModuleBaseUrls": [],
   "entryModuleId": "react",
   "scriptResources": {
   "react": {
   "type": "path",
   "path": "https://localhost:4321/temp/workbench-packages/react/umd/react.development.js"
   }
   }
   }
   },
   {
   "id": "aa0a46ec-1505-43cd-a44a-93f3a5aa460a",
   "manifestVersion": 2,
   "componentType": "Library",
   "alias": "react-dom",
   "version": "16.7.0",
   "loaderConfig": {
   "internalModuleBaseUrls": [],
   "entryModuleId": "react-dom",
   "scriptResources": {
   "react-dom": {
   "type": "path",
   "path": "https://localhost:4321/temp/workbench-packages/react-dom/umd/react-dom.development.js"
   },
   "react": {
   "type": "component",
   "id": "0d910c1c-13b9-4e1c-9aa4-b008c5e42d7d",
   "version": "16.7.0"
   }
   }
   }
   }
   ]
   </script>
   <script type="text/javascript" src="https://localhost:4321/temp/workbench-packages/@microsoft_sp-webpart-workbench/lib/api/workbenchInit.js"></script>
   <script type="text/javascript">
   // Clean up the "exports" object
   exports = undefined;
   </script>
   </head>
   <body>
    
   <script type="text/javascript">
   window.spModuleLoader.start(window.preloadedData);
   </script>
   <script src="//localhost:35729/livereload.js?snipver=1" async="" defer=""></script></body>
   </html>  
```

The Dev toolbar console will produce tons of errors:
```
sp-loader-assembly_default.js:5389 Can't load the application on this page. Use the browser Back button to retry. If the problem persists, contact the administrator of the site and give them the information in Technical Details.
SPStarter._consoleErrorHandleFailure @ sp-loader-assembly_default.js:5389
SPStarter.handleError @ sp-loader-assembly_default.js:5375
(anonymous) @ sp-loader-assembly_default.js:5267
Promise.catch (async)
SPStarter.start @ sp-loader-assembly_default.js:5262
(anonymous) @ workbench.html:69
sp-loader-assembly_default.js:5391 Correlation Id: 00000000-0000-4000-b000-000000000000
SPStarter._consoleErrorHandleFailure @ sp-loader-assembly_default.js:5391
SPStarter.handleError @ sp-loader-assembly_default.js:5375
(anonymous) @ sp-loader-assembly_default.js:5267
Promise.catch (async)
SPStarter.start @ sp-loader-assembly_default.js:5262
(anonymous) @ workbench.html:69
sp-loader-assembly_default.js:5394 Operation name: ClientError
SPStarter._consoleErrorHandleFailure @ sp-loader-assembly_default.js:5394
SPStarter.handleError @ sp-loader-assembly_default.js:5375
(anonymous) @ sp-loader-assembly_default.js:5267
Promise.catch (async)
SPStarter.start @ sp-loader-assembly_default.js:5262
(anonymous) @ workbench.html:69
sp-loader-assembly_default.js:5397 ***Platform failed to load. Id: "4df9bb86-ab0a-4aab-ab5f-48bf167048fb", name: "@microsoft/sp-application-base"
SPStarter._consoleErrorHandleFailure @ sp-loader-assembly_default.js:5397
SPStarter.handleError @ sp-loader-assembly_default.js:5375
(anonymous) @ sp-loader-assembly_default.js:5267
Promise.catch (async)
SPStarter.start @ sp-loader-assembly_default.js:5262
(anonymous) @ workbench.html:69
sp-loader-assembly_default.js:5398 CALL STACK: Error: ***Platform failed to load. Id: "4df9bb86-ab0a-4aab-ab5f-48bf167048fb", name: "@microsoft/sp-application-base"
    at error (https://localhost:4321/temp/workbench-packages/@microsoft_sp-loader/dist/sp-loader-assembly_default.js:5586:23)
    at https://localhost:4321/temp/workbench-packages/@microsoft_sp-loader/dist/sp-loader-assembly_default.js:5604:24
SPStarter._consoleErrorHandleFailure @ sp-loader-assembly_default.js:5398
SPStarter.handleError @ sp-loader-assembly_default.js:5375
(anonymous) @ sp-loader-assembly_default.js:5267
Promise.catch (async)
SPStarter.start @ sp-loader-assembly_default.js:5262
(anonymous) @ workbench.html:69
sp-loader-assembly_default.js:8383 Uncaught (in promise) Error
    at SPLoaderError.SPError [as constructor] (sp-loader-assembly_default.js:8390)
    at new SPLoaderError (sp-loader-assembly_default.js:4036)
    at Function.ErrorBuilder.buildErrorWithVerboseLog (sp-loader-assembly_default.js:3789)
    at Function.ErrorBuilder.buildLoadComponentError (sp-loader-assembly_default.js:3697)
    at sp-loader-assembly_default.js:6275
SPError @ sp-loader-assembly_default.js:8383
SPLoaderError @ sp-loader-assembly_default.js:4036
ErrorBuilder.buildErrorWithVerboseLog @ sp-loader-assembly_default.js:3789
ErrorBuilder.buildLoadComponentError @ sp-loader-assembly_default.js:3697
(anonymous) @ sp-loader-assembly_default.js:6275
Promise.then (async)
ComponentStore.storeComponent @ sp-loader-assembly_default.js:3616
loadComponent @ sp-loader-assembly_default.js:6215
_loop_1 @ sp-loader-assembly_default.js:6295
_loadComponentDependencies @ sp-loader-assembly_default.js:6318
_loadComponentImpl @ sp-loader-assembly_default.js:6252
_loadComponentRetryStrategy @ sp-loader-assembly_default.js:6225
(anonymous) @ sp-loader-assembly_default.js:6233
Promise.catch (async)
_loadComponentRetryStrategy @ sp-loader-assembly_default.js:6230
(anonymous) @ sp-loader-assembly_default.js:6233
Promise.catch (async)
_loadComponentRetryStrategy @ sp-loader-assembly_default.js:6230
loadComponent @ sp-loader-assembly_default.js:6211
SPSystemJsComponentLoader.loadComponent @ sp-loader-assembly_default.js:6596
BaseComponentLoader.loadComponentById @ sp-loader-assembly_default.js:5723
SPComponentLoaderProxy.loadComponentById @ sp-loader-assembly_default.js:3945
PlatformLoader._loadSpApplicationBase @ sp-loader-assembly_default.js:5655
PlatformLoader._startApplication @ sp-loader-assembly_default.js:5601
PlatformLoader.startApplication @ sp-loader-assembly_default.js:5581
BaseComponentLoader._startApplication @ sp-loader-assembly_default.js:5681
SPComponentLoaderProxy._startApplication @ sp-loader-assembly_default.js:3933
(anonymous) @ sp-loader-assembly_default.js:5242
Promise.then (async)
SPStarter.start @ sp-loader-assembly_default.js:5235
(anonymous) @ workbench.html:69
sp-loader-assembly_default.js:8383 Uncaught (in promise) Error
    at SPLoaderError.SPError [as constructor] (sp-loader-assembly_default.js:8390)
    at new SPLoaderError (sp-loader-assembly_default.js:4036)
    at Function.ErrorBuilder.buildErrorWithVerboseLog (sp-loader-assembly_default.js:3789)
    at Function.ErrorBuilder.buildLoadComponentError (sp-loader-assembly_default.js:3697)
    at sp-loader-assembly_default.js:6275
SPError @ sp-loader-assembly_default.js:8383
SPLoaderError @ sp-loader-assembly_default.js:4036
ErrorBuilder.buildErrorWithVerboseLog @ sp-loader-assembly_default.js:3789
ErrorBuilder.buildLoadComponentError @ sp-loader-assembly_default.js:3697
(anonymous) @ sp-loader-assembly_default.js:6275
Promise.then (async)
ComponentStore.storeComponent @ sp-loader-assembly_default.js:3616
loadComponent @ sp-loader-assembly_default.js:6215
SPSystemJsComponentLoader.loadComponent @ sp-loader-assembly_default.js:6596
BaseComponentLoader.loadComponentById @ sp-loader-assembly_default.js:5723
SPComponentLoaderProxy.loadComponentById @ sp-loader-assembly_default.js:3945
PlatformLoader._loadSpApplicationBase @ sp-loader-assembly_default.js:5655
PlatformLoader._startApplication @ sp-loader-assembly_default.js:5601
PlatformLoader.startApplication @ sp-loader-assembly_default.js:5581
BaseComponentLoader._startApplication @ sp-loader-assembly_default.js:5681
SPComponentLoaderProxy._startApplication @ sp-loader-assembly_default.js:3933
(anonymous) @ sp-loader-assembly_default.js:5242
Promise.then (async)
SPStarter.start @ sp-loader-assembly_default.js:5235
(anonymous) @ workbench.html:69
sp-loader-assembly_default.js:8383 Uncaught (in promise) Error
    at SPLoaderError.SPError [as constructor] (sp-loader-assembly_default.js:8390)
    at new SPLoaderError (sp-loader-assembly_default.js:4036)
    at Function.ErrorBuilder.buildErrorWithVerboseLog (sp-loader-assembly_default.js:3789)
    at Function.ErrorBuilder.buildLoadComponentError (sp-loader-assembly_default.js:3697)
    at sp-loader-assembly_default.js:6275
SPError @ sp-loader-assembly_default.js:8383
SPLoaderError @ sp-loader-assembly_default.js:4036
ErrorBuilder.buildErrorWithVerboseLog @ sp-loader-assembly_default.js:3789
ErrorBuilder.buildLoadComponentError @ sp-loader-assembly_default.js:3697
(anonymous) @ sp-loader-assembly_default.js:6275
Promise.then (async)
ComponentStore.storeComponent @ sp-loader-assembly_default.js:3616
loadComponent @ sp-loader-assembly_default.js:6215
_loop_1 @ sp-loader-assembly_default.js:6295
_loadComponentDependencies @ sp-loader-assembly_default.js:6318
_loadComponentImpl @ sp-loader-assembly_default.js:6252
_loadComponentRetryStrategy @ sp-loader-assembly_default.js:6225
loadComponent @ sp-loader-assembly_default.js:6211
_loop_1 @ sp-loader-assembly_default.js:6295
_loadComponentDependencies @ sp-loader-assembly_default.js:6318
_loadComponentImpl @ sp-loader-assembly_default.js:6252
_loadComponentRetryStrategy @ sp-loader-assembly_default.js:6225
loadComponent @ sp-loader-assembly_default.js:6211
SPSystemJsComponentLoader.loadComponent @ sp-loader-assembly_default.js:6596
BaseComponentLoader.loadComponentById @ sp-loader-assembly_default.js:5723
SPComponentLoaderProxy.loadComponentById @ sp-loader-assembly_default.js:3945
PlatformLoader._loadSpApplicationBase @ sp-loader-assembly_default.js:5655
PlatformLoader._startApplication @ sp-loader-assembly_default.js:5601
PlatformLoader.startApplication @ sp-loader-assembly_default.js:5581
BaseComponentLoader._startApplication @ sp-loader-assembly_default.js:5681
SPComponentLoaderProxy._startApplication @ sp-loader-assembly_default.js:3933
(anonymous) @ sp-loader-assembly_default.js:5242
Promise.then (async)
SPStarter.start @ sp-loader-assembly_default.js:5235
(anonymous) @ workbench.html:69
sp-loader-assembly_default.js:8383 Uncaught (in promise) Error
    at SPLoaderError.SPError [as constructor] (sp-loader-assembly_default.js:8390)
    at new SPLoaderError (sp-loader-assembly_default.js:4036)
    at Function.ErrorBuilder.buildErrorWithVerboseLog (sp-loader-assembly_default.js:3789)
    at Function.ErrorBuilder.buildLoadComponentError (sp-loader-assembly_default.js:3697)
    at sp-loader-assembly_default.js:6275
SPError @ sp-loader-assembly_default.js:8383
SPLoaderError @ sp-loader-assembly_default.js:4036
ErrorBuilder.buildErrorWithVerboseLog @ sp-loader-assembly_default.js:3789
ErrorBuilder.buildLoadComponentError @ sp-loader-assembly_default.js:3697
(anonymous) @ sp-loader-assembly_default.js:6275
Promise.then (async)
ComponentStore.storeComponent @ sp-loader-assembly_default.js:3616
loadComponent @ sp-loader-assembly_default.js:6215
_loop_1 @ sp-loader-assembly_default.js:6295
_loadComponentDependencies @ sp-loader-assembly_default.js:6318
_loadComponentImpl @ sp-loader-assembly_default.js:6252
_loadComponentRetryStrategy @ sp-loader-assembly_default.js:6225
loadComponent @ sp-loader-assembly_default.js:6211
SPSystemJsComponentLoader.loadComponent @ sp-loader-assembly_default.js:6596
BaseComponentLoader.loadComponentById @ sp-loader-assembly_default.js:5723
SPComponentLoaderProxy.loadComponentById @ sp-loader-assembly_default.js:3945
PlatformLoader._loadSpApplicationBase @ sp-loader-assembly_default.js:5655
PlatformLoader._startApplication @ sp-loader-assembly_default.js:5601
PlatformLoader.startApplication @ sp-loader-assembly_default.js:5581
BaseComponentLoader._startApplication @ sp-loader-assembly_default.js:5681
SPComponentLoaderProxy._startApplication @ sp-loader-assembly_default.js:3933
(anonymous) @ sp-loader-assembly_default.js:5242
Promise.then (async)
SPStarter.start @ sp-loader-assembly_default.js:5235
(anonymous) @ workbench.html:69
sp-loader-assembly_default.js:5586 Uncaught (in promise) Error: ***Platform failed to load. Id: "4df9bb86-ab0a-4aab-ab5f-48bf167048fb", name: "@microsoft/sp-application-base"
    at error (sp-loader-assembly_default.js:5586)
    at sp-loader-assembly_default.js:5604
error @ sp-loader-assembly_default.js:5586
(anonymous) @ sp-loader-assembly_default.js:5604
```

Running the exact same solutions in my online tenant workbench works. 

I've tried uninstalling everything and re-installing everything, short of a complete OS reinstall.

#### Steps to Reproduce

1. Create a new solution or download an existing one from sp-dev-fx-webparts
2. If downloading a solution, run `npm install`
3. run `gulp serve`
4. Watch local workbench load and display nothing.

I hope I'm not the only one with this issue. Well, actually, I wouldn't wish this issue on anyone :-)

**Comment by andrewconnell** [comment]
Any errors in the console? What version of SPFx? What version of Node?

**Comment by hugoabernier** [comment]
Errors in console are listed in the **Observed Behavior** above.

Node: 10.16.0 LTS. Re-installed it yesterday, just in case.
SPFx: 1.8.2

**Comment by andrewconnell** [comment]
Sorry... when I said "console" I meant the command prompt / terminal. Not the browser JS console...

**Comment by hugoabernier** [comment]
Ah, makes more sense. No, nothing.

**Comment by andrewconnell** [comment]
There's got to be _something_ in the command prompt/terminal...

**Comment by hugoabernier** [comment]
lol, yes there is *something*, I'll grab it later today. You asked if there were error messages and there are none.

Thanks again for helping with this.

**Comment by patmill** [comment]
Also - what happens when you hit the hosted workbench?

**Comment by hugoabernier** [comment]
Apologies in advance for the lengthy reply. Here is the output of the console:
```shell
λ gulp serve                                                                                                                          
Build target: DEBUG                                                                                                                   
[20:20:10] Using gulpfile C:\github\WebPartDesignSeries\WebPartTitles\gulpfile.js                                                     
[20:20:10] Starting gulp                                                                                                              
[20:20:10] Starting 'serve'...                                                                                                        
[20:20:10] Starting subtask 'configure-sp-build-rig'...                                                                               
[20:20:10] Finished subtask 'configure-sp-build-rig' after 6.8 ms                                                                     
[20:20:10] Starting subtask 'spfx-serve'...                                                                                           
[20:20:10] Starting server...                                                                                                         
Starting api server on port 5432.                                                                                                     
Registring api: /workbench                                                                                                            
Registring api: */*                                                                                                                   
[20:20:10] Finished subtask 'spfx-serve' after 393 ms                                                                                 
[20:20:10] Starting subtask 'pre-copy'...                                                                                             
[20:20:10] Finished subtask 'pre-copy' after 54 ms                                                                                    
[20:20:10] Starting subtask 'copy-static-assets'...                                                                                   
[20:20:10] Starting subtask 'sass'...                                                                                                 
[20:20:10] Server started https://localhost:4321                                                                                      
[20:20:10] LiveReload started on port 35729                                                                                           
[20:20:10] Running server                                                                                                             
[20:20:10] Opening https://localhost:5432/workbench using the default OS app                                                          
[20:20:10] Finished subtask 'copy-static-assets' after 48 ms                                                                          
[20:20:11] Finished subtask 'sass' after 258 ms                                                                                       
[20:20:11] Starting subtask 'tslint'...                                                                                               
[20:20:12] [tslint] tslint version: 5.12.1                                                                                            
[20:20:12] Starting subtask 'tsc'...                                                                                                  
[20:20:12] [tsc] typescript version: 2.9.2                                                                                            
  Request: [::1] '/workbench'                                                                                                         
  Request: '/temp/workbench.html'                                                                                                     
  Request: '/temp/manifests.js'                                                                                                       
  Request: '/temp/workbench-packages/@microsoft_sp-loader/dist/sp-loader-assembly_default.js'                                         
  Request: '/temp/workbench-packages/@microsoft_sp-webpart-workbench/lib/api/workbenchInit.js'                                        
  Request: '/users/hugoa/onedrive/github/WebPartDesignSeries/WebPartTitles/node_modules/@microsoft/decorators/dist/decorators.js'     
  Request: '/users/hugoa/onedrive/github/WebPartDesignSeries/WebPartTitles/node_modules/@microsoft/decorators/dist/decorators.js'     
  Request: '/users/hugoa/onedrive/github/WebPartDesignSeries/WebPartTitles/node_modules/@microsoft/decorators/dist/decorators.js'     
  Request: '/users/hugoa/onedrive/github/WebPartDesignSeries/WebPartTitles/node_modules/@microsoft/decorators/dist/decorators.js'     
  Request: '/users/hugoa/onedrive/github/WebPartDesignSeries/WebPartTitles/node_modules/@microsoft/decorators/dist/decorators.js'     
  Request: '/users/hugoa/onedrive/github/WebPartDesignSeries/WebPartTitles/node_modules/@microsoft/decorators/dist/decorators.js'     
  Request: '/users/hugoa/onedrive/github/WebPartDesignSeries/WebPartTitles/node_modules/@microsoft/decorators/dist/decorators.js'     
  Request: '/users/hugoa/onedrive/github/WebPartDesignSeries/WebPartTitles/node_modules/@microsoft/decorators/dist/decorators.js'     
  Request: '/users/hugoa/onedrive/github/WebPartDesignSeries/WebPartTitles/node_modules/@microsoft/decorators/dist/decorators.js'     
  Request: '/users/hugoa/onedrive/github/WebPartDesignSeries/WebPartTitles/node_modules/@microsoft/decorators/dist/decorators.js'     
  Request: '/users/hugoa/onedrive/github/WebPartDesignSeries/WebPartTitles/node_modules/@microsoft/decorators/dist/decorators.js'     
  Request: '/users/hugoa/onedrive/github/WebPartDesignSeries/WebPartTitles/node_modules/@microsoft/decorators/dist/decorators.js'     
  Request: '/users/hugoa/onedrive/github/WebPartDesignSeries/WebPartTitles/node_modules/@microsoft/decorators/dist/decorators.js'     
  Request: '/users/hugoa/onedrive/github/WebPartDesignSeries/WebPartTitles/node_modules/@microsoft/decorators/dist/decorators.js'     
  Request: '/users/hugoa/onedrive/github/WebPartDesignSeries/WebPartTitles/node_modules/@microsoft/decorators/dist/decorators.js'     
  Request: '/users/hugoa/onedrive/github/WebPartDesignSeries/WebPartTitles/node_modules/@microsoft/decorators/dist/decorators.js'     
  Request: '/users/hugoa/onedrive/github/WebPartDesignSeries/WebPartTitles/node_modules/@microsoft/decorators/dist/decorators.js'     
  Request: '/users/hugoa/onedrive/github/WebPartDesignSeries/WebPartTitles/node_modules/@microsoft/decorators/dist/decorators.js'     
[20:20:14] Finished subtask 'tsc' after 1.99 s                                                                                        
[20:20:14] Finished subtask 'tslint' after 3.31 s                                                                                     
[20:20:14] Starting subtask 'post-copy'...                                                                                            
[20:20:14] Finished subtask 'post-copy' after 852 μs                                                                                  
[20:20:14] Starting subtask 'collectLocalizedResources'...                                                                            
[20:20:14] Finished subtask 'collectLocalizedResources' after 3.37 ms                                                                 
[20:20:14] Starting subtask 'configure-webpack'...                                                                                    
[20:20:15] Finished subtask 'configure-webpack' after 1.34 s                                                                          
[20:20:15] Starting subtask 'webpack'...                                                                                              
[20:20:18] Finished subtask 'webpack' after 2.73 s                                                                                    
[20:20:18] Starting subtask 'configure-webpack-external-bundling'...                                                                  
[20:20:18] Finished subtask 'configure-webpack-external-bundling' after 950 μs                                                        
[20:20:18] Starting subtask 'copy-assets'...                                                                                          
[20:20:18] Finished subtask 'copy-assets' after 16 ms                                                                                 
[20:20:18] Starting subtask 'write-manifests'...                                                                                      
[20:20:20] Finished subtask 'write-manifests' after 1.52 s                                                                            
[20:20:20] Starting subtask 'reload'...                                                                                               
[20:20:20] Finished subtask 'reload' after 1.5 ms                                                                                     
  Request: '/temp/workbench.html'                                                                                                     
  Request: '/temp/manifests.js'                                                                                                       
  Request: '/temp/workbench-packages/@microsoft_sp-loader/dist/sp-loader-assembly_default.js'                                         
  Request: '/temp/workbench-packages/@microsoft_sp-webpart-workbench/lib/api/workbenchInit.js'                                        
  Request: '/users/hugoa/onedrive/github/WebPartDesignSeries/WebPartTitles/node_modules/@microsoft/decorators/dist/decorators.js'     
  Request: '/users/hugoa/onedrive/github/WebPartDesignSeries/WebPartTitles/node_modules/@microsoft/decorators/dist/decorators.js'     
  Request: '/users/hugoa/onedrive/github/WebPartDesignSeries/WebPartTitles/node_modules/@microsoft/decorators/dist/decorators.js'     
  Request: '/users/hugoa/onedrive/github/WebPartDesignSeries/WebPartTitles/node_modules/@microsoft/decorators/dist/decorators.js'     
  Request: '/users/hugoa/onedrive/github/WebPartDesignSeries/WebPartTitles/node_modules/@microsoft/decorators/dist/decorators.js'     
  Request: '/users/hugoa/onedrive/github/WebPartDesignSeries/WebPartTitles/node_modules/@microsoft/decorators/dist/decorators.js'     
  Request: '/users/hugoa/onedrive/github/WebPartDesignSeries/WebPartTitles/node_modules/@microsoft/decorators/dist/decorators.js'     
  Request: '/users/hugoa/onedrive/github/WebPartDesignSeries/WebPartTitles/node_modules/@microsoft/decorators/dist/decorators.js'     
  Request: '/users/hugoa/onedrive/github/WebPartDesignSeries/WebPartTitles/node_modules/@microsoft/decorators/dist/decorators.js'     
  Request: '/users/hugoa/onedrive/github/WebPartDesignSeries/WebPartTitles/node_modules/@microsoft/decorators/dist/decorators.js'     
  Request: '/users/hugoa/onedrive/github/WebPartDesignSeries/WebPartTitles/node_modules/@microsoft/decorators/dist/decorators.js'     
  Request: '/users/hugoa/onedrive/github/WebPartDesignSeries/WebPartTitles/node_modules/@microsoft/decorators/dist/decorators.js'     
  Request: '/users/hugoa/onedrive/github/WebPartDesignSeries/WebPartTitles/node_modules/@microsoft/decorators/dist/decorators.js'     
  Request: '/users/hugoa/onedrive/github/WebPartDesignSeries/WebPartTitles/node_modules/@microsoft/decorators/dist/decorators.js'     
  Request: '/users/hugoa/onedrive/github/WebPartDesignSeries/WebPartTitles/node_modules/@microsoft/decorators/dist/decorators.js'     
  Request: '/users/hugoa/onedrive/github/WebPartDesignSeries/WebPartTitles/node_modules/@microsoft/decorators/dist/decorators.js'     
  Request: '/users/hugoa/onedrive/github/WebPartDesignSeries/WebPartTitles/node_modules/@microsoft/decorators/dist/decorators.js'     
  Request: '/users/hugoa/onedrive/github/WebPartDesignSeries/WebPartTitles/node_modules/@microsoft/decorators/dist/decorators.js'     
```

And the screenshot of the browser window:
![Blank browser](https://user-images.githubusercontent.com/13972467/60931349-40070f80-a287-11e9-8b65-36306e53fd48.png)

And here is a screenshot of the hosted workbench (same `gulp serve` instance):
![Not so blank](https://user-images.githubusercontent.com/13972467/60931374-62009200-a287-11e9-9022-29a847601a7d.png)

**Comment by BassemNKhalil** [comment]
Hey @hugoabernier, any chance you're using SPFx 1.8.2 but targeting on-prem?
I'm seeing similar results in #4485 but only for SP16 targeted web-part, not SPO ones.

**Comment by hugoabernier** [comment]
Hey @BassemNKhalil , thanks for your comment, but no. 

I can move the entire solution to another machine and it works, just this one workstation always produces a blank workbench.

**Comment by andrewconnell** [comment]
@hugoabernier said:
> I can move the entire solution to another machine and it works

In that case, let's see what the difference is. First, I assume you've tried deleting the **node_modules** folder & any package lock files in the project, then reinstalling all dependencies with **npm install** (or the equivalent package manager). If that still doesn't work, please report back the following comparison run on both machines of JUST what's different.

- OS including version
- browser including version
- Node.js & NPM version (`node -v` & `npm -v`)
- NPM packages globally installed (`npm list -g --depth=0`)
- `PATH` contents for all environment variables

**Comment by hugoabernier** [comment]
Sorry @andrewconnell , the issue isn't with a specific solution, the issue is with *every* solution on that particular workstation. I can take any SPFx solution from any repo and I get the same behavior.

I tried deleting the node_modules, reinstalling node.js, NPM, and everything else I could think of -- short of re-installing the OS.

I'll try to run a compare between a few machines and see what's different, and will report back.

Thanks!

**Comment by shaneiadt** [comment]
I spent all day tackling this issue, installing multiple version of node & spfx with no change. Running with elevated permissions no dice!

But I do have a solution that worked using the below environment setup:

```
Windows 10 Pro - 10.0.18362
node v10.16.3
@microsoft/generator-sharepoint@1.9.1
gulp@4.0.2
yo@3.1.0
```

I was recently playing around with Docker & **Postgres**.

The default port **Postgres** runs on is you guessed it **5432**!! So in a last ditch effort I attempted to updated the port value in my `config/serve.json` to **5433**:

```
{
  "$schema": "https://developer.microsoft.com/json-schemas/core-build/serve.schema.json",
  "port": 4321,
  "https": true,
  "initialPage": "https://localhost:5433/workbench",
  "api": {
    "port": 5433,
    "entryPath": "node_modules/@microsoft/sp-webpart-workbench/lib/api/"
  }
}
```
BOOM 💥 

This worked for me! I'm hoping saves someone else a days hair pulling :sweat:

**Comment by andrewconnell** [comment]
@shaneiadt Yours is a known scenario... if you looked at the console output, there was an error from the toolchain listed saying it couldn't attach to the port because it was already in use.

This thread contains the console output ([ref)(https://github.com/SharePoint/sp-dev-docs/issues/4261#issuecomment-509858713)) which doesn't have this listed, so your solution doesn't apply in this scenario.

**Comment by johnguy0** [comment]
@hugoabernier Can you provide the details @andrewconnell requested in the thread? Is this only happening to one particular machine? Does this continue to happen if you upgrade the generator to 1.10.0?

**Comment by hugoabernier** [comment]
@johnguy0 it continues to happen even after upgrading. Again, this issue occurs with every solution, regardless of the SPFx version on that workstation.

**Comment by johnguy0** [comment]
@hugoabernier Without any further details about the machine and environment it's difficult for us to help. We cannot reproduce the issue.

**Comment by hugoabernier** [comment]
OS: Windows 10 v2004 (OS Build 19546.1000)
Browser: Chrome, Edge, Edge Chromium
Node: v8.16.0 (tried upgrading and downgrading, didn't matter)
NPM: 6.4.1
NPM List -g: 
+-- @microsoft/generator-sharepoint@1.10.0
+-- @pnp/office365-cli@2.5.0-beta.65b2c66
+-- dts-gen@0.5.7
+-- generator-office@1.4.0
+-- generator-teams@2.11.1
+-- grunt-cli@1.3.2
+-- gulp@4.0.2
+-- jison@0.4.18
+-- js-to-ts-converter@0.16.2
+-- npm-check-updates@3.1.10
`-- yo@3.1.1

PATH:
PATH=C:\Users\hugoa\AppData\Local\cmder\bin;C:\Users\hugoa\AppData\Local\cmder\vendor\bin;C:\Users\hugoa\AppData\Local\cmder\vendor\conemu-maximus5\ConEmu\Scripts;C:\Users\hugoa\AppData\Local\cmder\vendor\conemu-maximus5;C:\Users\hugoa\AppData\Local\cmder\vendor\conemu-maximus5\ConEmu;C:\Program Files\Microsoft MPI\Bin\;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;C:\Program Files\dotnet\;C:\Program Files\Microsoft SQL Server\130\Tools\Binn\;C:\Program Files\Git\cmd;C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\170\Tools\Binn\;C:\Program Files\PowerShell\6\;C:\ProgramData\chocolatey\bin;C:\Program Files\Microsoft VS Code Insiders\bin;C:\Program Files\nodejs\;C:\Users\hugoa\AppData\Local\Programs\Python\Python37-32\Scripts\;C:\Users\hugoa\AppData\Local\Programs\Python\Python37-32\;C:\Users\hugoa\AppData\Local\Microsoft\PowerAppsCLI\;C:\Users\hugoa\AppData\Local\Microsoft\WindowsApps;C:\Users\hugoa\AppData\Local\Programs\Microsoft VS Code\bin;C:\Users\hugoa\AppData\Local\Programs\Azure Data Studio\bin;C:\Users\hugoa\AppData\Roaming\npm;C:\Program Files\dotnet;C:\Users\hugoa\.dotnet\tools;%USERPROFILE%\AppData\Local\Microsoft\WindowsApps;C:\Program Files\Git\cmd;C:\Program Files\Git\mingw64\bin;C:\Program Files\Git\usr\bin;C:\Users\hugoa\AppData\Local\cmder

I hope this helps.

**Comment by johnguy0** [comment]
@hugoabernier Thanks! We'll do our best to try to reproduce the issue.

**Comment by andrewconnell** [comment]
@hugoabernier I can't repro this, unfortunately. I've one had one of my customers experience this & we could never figure out what it was. 

But we fixed it... I don't know what fixed it, but this worked: **start over**.

Uninstall everything, specifically Node & globally installed packages. Make sure it was completely gone in the `PATH` environment variable, make sure the folder where the global NPM packages were installed is completely gone as well. Empty trash & reboot.

Now... get a new Node install (I'd rec v10.* if you're using SPFx v1.10), then Yeoman, Gulp & the SPFx generator. Now create a new project and give it a shot.

**Comment by hugoabernier** [comment]
Well, here is a recent development. 

I did not change anything in my environment. 

I created a web part using the PnP Yeoman generator instead of the Yeoman generator... Then ran `gulp serve` and the workbench worked.

Then I created a new web part using the Yeoman generator and ran the workbench and... it worked!

I don't like bugs that go away magically, because it just means that the condition that caused the bug has momentarily disappeared... I'll continue monitoring the issue and will report if the issue re-appears.
